File: snippets/js/array/cut/cut.js
var array = [5, 4, 3, 2, 1];
var start = 1;  // 0: first item, 1: second item...
var count = 3;
console.log(array);     // [ 5, 4, 3, 2, 1 ]
var arrayCut = array.splice(start, count);  // ≡
console.log(array);     // [ 5, 1 ]
console.log(arrayCut);  // [ 4, 3, 2 ]

File: snippets/py/array/cut/cut.py
array = [5, 4, 3, 2, 1]
start = 1  # 0: first item, 1: second item...
count = 3
print(array)     # [ 5, 4, 3, 2, 1 ]
arrayCut = array[start:start + count]  # ≡
del array[start:start + count]         # ≡
print(array)     # [ 5, 1 ]
print(arrayCut)  # [ 4, 3, 2 ]